etc/ghostwriter.profile

```diff
@@ -0,0 +1,50 @@
+# Firejail profile for ghostwriter
+# Description: Cross-platform, aesthetic, distraction-free Markdown editor.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include ghostwriter.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/ghostwriter
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+
+mkdir ${HOME}/.config/ghostwriter
+whitelist ${HOME}/.config/ghostwriter
+include whitelist-common.inc
+
+apparmor
+caps.drop all
+netfilter
+#no3d
+#nodbus
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,netlink
+seccomp
+shell none
+tracelog
+machine-id
```

Comment:
Please move `machine-id` between `caps.drop.all` and `netfilter`.

Reply:
Fixed, not pushed yet.

Reply:
Thanks, but I still think a hint in `CONTRIBUTING.md` is good for newcomers.